internal/ui/table.go

```diff
@@ -448,6 +476,48 @@ func (t *Table) SortColCmd(name string, asc bool) func(evt *tcell.EventKey) *tce
 	}
 }
 
+// SortColChange changes on which column to sort
+func (t *Table) SortColChange(direction SortChange) func(evt *tcell.EventKey) *tcell.EventKey {
+	return func(evt *tcell.EventKey) *tcell.EventKey {
+		sortCol := t.sortCol.Name
+		sortColIdx := -1
+		newSortColIdx := -1
+		prevSortColIdx := -1
+		header := t.GetModel().Peek().GetHeader()
```

Comment:
This is not ideal as we realize the model twice here and later in refresh which would be expensive.
So I think we should pass in the model to something like doRefresh(*model1.Data) vs calling Refresh once the sort is set.

Reply:
Also this should call setMSort(true) to tell the table manual sorting is now in effect.